Automattic/wp-calypso
# Create and add simple product card in the new pricing page.

#### Proposed Changes

* Implement a reusable product simple card component. Note below:
  * The 'more' button does not work at the moment and should be implemented once the lightbox product details are ready.
  * There is still issue w/ the styling for the `ItemPrice` component while it is on fetching state but this should also be fix in another ticket.
  * Flow for already activated product simple card should be handled in another ticket.
  
* Render product simple card in the Product page with ordering base on the figma design. Note that `Jetpack social` is still missing as the constants and data are not yet put in place. I decided not to include it for now to avoid making this PR too big while tackling more files. This should be resolve in another ticket for adding the new data. 

#### Testing Instructions



* Boot up this PR
  * Run `git fetch && git checkout add/pricing-page-product-card`
  * Run `yarn start-jetpack-cloud`
* Go to http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1 or use the jetpack cloud live link and goto `/pricing?flags=jetpack/pricing-page-rework-v1`.
* Confirm that the 'All products' section is visible with the simple card.

<img width="1211" alt="Screen Shot 2022-08-30 at 7 29 51 PM" src="https://user-images.githubusercontent.com/56598660/187425699-2a0b91c4-90f6-4c62-a272-3db7b4733f71.png">

* Confirm if the simple card is responsive and work as expected on any viewpoint.
<img width="464" alt="Screen Shot 2022-08-30 at 7 30 05 PM" src="https://user-images.githubusercontent.com/56598660/187425842-39b3b287-7f2d-4fc4-9374-0a9ead863a76.png">

* Confirm the 'Get' button works and redirect to correct checkout page.
* Confirm the 'Get' button for the Jetpack CRM redirect to the correct CRM pricing page.

#### Pre-merge Checklist






Related to 1202796695664022-as-1202796695664111

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202796695664111
